api/apps/graphql/index.js

```diff
@@ -12,18 +12,19 @@ import * as resolvers from 'api/apps/graphql/resolvers.js'
 
 const app = module.exports = express()
 
-const relatedArticles = {
+const metaFields = {
   relatedArticlesCanvas: array().items(object(Article.inputSchema)).meta({
     name: 'RelatedArticlesCanvas',
     resolve: resolvers.relatedArticlesCanvas
   }),
   relatedArticlesPanel: array().items(object(Article.inputSchema)).meta({
     name: 'RelatedArticlesPanel',
     resolve: resolvers.relatedArticlesPanel
-  })
+  }),
+  is_super_sub_article: boolean()
```

Comment:
Now that we're not including `is_super_sub_article` directly in the Joi schema, this makes it available in the graphql schema manually.